### ISSUE TITLE
iframes in panel mode lose styling due to card-wrapping

### ISSUE THREAD

**Issue by zaneclaes** [gadgetchnnel/lovelace-card-templater#17]
When `lovelace-card-templater` is used to create a card of `type: iframe` which is subsequently used in a `panel`, the iframe only consumes 50% of the screen height. Simply using the iframe card directly causes the iframe to consume exactly the full viewport area. This is due to the way that the templating card wraps the iframe with its own card, but does not pass through the necessary CSS styles.

Works:
```
url: 'https://audio.snowy-cabin.com'
type: iframe
```

Does not work:
```
type: 'custom:card-templater'
card:
  url: 'https://audio.snowy-cabin.com'
  type: iframe
```

**Comment by gadgetchnnel** [comment]
@zaneclaes I have just released a new version, 0.0.9, which a few fixes including one for this issue. Can you please give it a try and let me know if it works for you?

**Comment by zaneclaes** [comment]
It didn't appear to work at first, but after a moment or two it expanded to fill the viewport. I'm guessing this was a JS event hook implementation, thus the delay. But LGTM! Thanks.
